example/selection.js

```diff
@@ -12,29 +12,239 @@ import {
 
 const params = {
 
+	/** Selection tool: lasso (freehand shape) or box. */
 	toolMode: 'lasso',
+	/**
+	 * How triangles are marked for selection:
+	 * - intersection: if any part of the triangle is within the selection shape.
+	 * - centroid: if the center of the triangle is within the selection shape.
+	 * - centroid-visible: if the center of the triangle is within the selection shape and the triangle is visible.
+	 */
 	selectionMode: 'intersection',
+	/** Show selected triangles during the drag or only after the selection is completed. */
 	liveUpdate: false,
+	/** Select the whole mesh if one of its triangles was intersected. */
 	selectModel: false,
+	/** How to render the object: with solid material or edges only (wireframe). */
 	wireframe: false,
+	/** Use the BVH to speed up computations. */
 	useBoundsTree: true,
 
+	/** Show the boxes of the BVH. */
 	displayHelper: false,
+	/** The depth of BVH boxes to display, needs `displayHelper: true`. */
 	helperDepth: 10,
+	/** Continuously rotate the object. */
 	rotate: true,
 
 };
 
+/** Abstract class representing a selection using a pointer. */
+class Selection {
```

Comment:
I think we should move these classes to their own files in `/examples/src` so this file is cleaned up a bit and these classes are easier to reuse. Ie "Selection.js" can house Selection, LassoSelection, and BoxSelection.

Reply:
Yep, sounds good! I just thought that it was a bit of an implicit goal for all examples to be self-contained

Reply:
Done!